code/__HELPERS/icons.dm

```diff
@@ -981,9 +981,10 @@ GLOBAL_LIST_EMPTY(friendly_animal_types)
 	var/static/list/humanoid_icon_cache = list()
 	if(icon_id && humanoid_icon_cache[icon_id])
 		return humanoid_icon_cache[icon_id]
-
-	var/mob/living/carbon/human/dummy/body = generate_or_wait_for_human_dummy(dummy_key)
```

Comment:
need to put this in a comment somewhere so we know what the default was.

Reply:
why not just right click -> open previous revision? this stuff is stored anyways

Reply:
because when we fix conflicts from upstream its hard to do that from the review screen; we cant really view those revisions, nor KNOW to, if its not labelled

Reply:
so just having SKYRAT EDIT means maintainers know 'this wasnt the original, do we want this to change to TG' without having to search every changed line from upstream

Reply:
> because when we fix conflicts from upstream its hard to do that from the review screen; we cant really view those revisions, nor KNOW to, if its not labelled

this is a good point, I will update this to do so